docs/docs/en/guide/resource/file-manage.md

```diff
@@ -0,0 +1,68 @@
+# File Management
+
+When third party jars are used in the scheduling process or user defined scripts are required, these can be created from this page. The types of files that can be created include: `txt/log/sh/conf/py/java` and so on. Files can be edited, renamed, downloaded and deleted.
```

Comment:
should we use common for `txt/log/sh/conf/py/java` like txt, log, sh, conf, py, java instead of using code block?